rpcs3/Emu/Cell/lv2/sys_net.cpp

```diff
@@ -256,6 +256,14 @@ lv2_socket::lv2_socket(utils::serial& ar, lv2_socket_type _type)
 	// Try to match structure between different platforms
 	ar.pos += 8;
 #endif
+
+	[[maybe_unused]] const s32 version = GET_SERIALIZATION_VERSION(lv2_net);
```

Comment:
```suggestion
	const s32 version = GET_SERIALIZATION_VERSION(lv2_net);
```

Reply:
This is for future proofing with future savestate changes, which may or may not need to reference version. So this line wouldn't need to be changed as time passes.
This is the practice in other places as well.

Reply:
this is premature and not relevant to the current code, so it should not be there at all.